sound/soc/sof/topology.h

```diff
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2025 Intel Corporation
+ */
+
+#ifndef __SOUND_SOC_SOF_TOPOLOGY_H
+#define __SOUND_SOC_SOF_TOPOLOGY_H
```

Comment:
I would rather create `card-tplg-ops.h` instead.

We might (_might_) be able to extend this to be used by the fw-file-profile.c and cut the reliance on the monolithic topology file even if we only expect to use a generic one.

But, but, I guess we can create a generic sof-<plat>-sdca-full-card.tplg and use this single one for all sdca setups...?